packages/manager/apps/veeam-backup/src/pages/listing/DatagridCell.component.tsx

```diff
@@ -115,17 +115,11 @@ export const RegionCell = ({
   currentState,
   isDashboard,
 }: VeeamBackupWithIam & { isDashboard?: boolean }): JSX.Element => {
-  const { flattenData, isLoading } = useRegions();
-  const label = flattenData?.find((r) => r.region === currentState?.region)
-    ?.location;
-
-  if (isLoading) {
-    return <OsdsSkeleton />;
-  }
-  return isDashboard ? (
-    <Description>{label}</Description>
-  ) : (
-    <DataGridTextCell>{label}</DataGridTextCell>
+  const Wrapper = isDashboard ? Description : DataGridTextCell;
```

Comment:
you could remove the the dashboard use-case as this component is intended as a Datagrid cell component. The common part is already handled by the `<Region />` component anyway